### PR TITLE
Use zeep instead of suds

### PR DESCRIPTION
**Pull request by sga-odoo**
Task: https://www.odoo.com/web?#id=41696&model=project.task&view_type=form&menu_id=4720
PAD: https://pad.odoo.com/p/r.9ed49d234bde1f09777c589174e235b5

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/base_vat_autocomplete/models/res_partner.py [hunk 1]
```diff
@@ -6,7 +6,8 @@
 
 from odoo import api, models
 
-from suds.client import Client
+from zeep import Client
```
[the remark this answers is not in the document]
Comment (sga-odoo) [reply addons/base_vat_autocomplete/models/res_partner.py]: Hey, zeep return direct client object so it is zeep not zeep.client . Ref: https://python-zeep.readthedocs.io/en/master/

[the remark this answers is not in the document]
Comment (mreficent) [reply addons/base_vat_autocomplete/models/res_partner.py]: ok

**Comment by gustavovalverde** [comment]
Is this hitting 12.0?

**Comment by tivisse** [comment]
@robodoo r+

**Pull request closed without merging**